### DISCUSSION TITLE
How does ffcv work?

### DISCUSSION THREAD

**Discussion by pdd-99** [libffcv/ffcv#138] [General]
I have already installed and played with ffcv for a while and now I want to dig a bit deeper. Just how ffcv work?

**Comment by GuillaumeLeclerc** [comment]
Hi @pdd-99!

I'll move this issue to the conversation tab as it seems more appropriate.

What would you like to know in particular ? (For an overview our README). Note that we plan on releasing a paper soon so if you want an in depth answer you might have to wait for that.

**Comment by alexmyczko** [comment]
I guess https://github.com/libffcv/ffcv/discussions/47 answers part of the question?
